vars/all_var.yml

```diff
@@ -96,38 +33,11 @@
   # False in IMPL / PROD
   django_debug: "{{ env_django_debug }}"
 
-  # The value of your django project's STATIC_ROOT settings.
-  # This will be the directory that django's `collectstatic` management command
-  # copies static files to, and it must be an absolute path. The default value
-  # here assumes that STATIC_ROOT is set in your settings.py like so:
-  # STATIC_ROOT = os.path.abspath(os.path.join(BASE_DIR, 'static'))
-  static_root: "{{ install_root }}/{{ project_name }}/{{ env_static_root_dir }}"
-
   # application owner: pyapps
   app_owner: "pyapps"
   # application group: nginx
   app_group: "www-data"
 
-  ### External Internet Domain Name for the platform
-  external_dns_name: "{{ env_external_dns_name }}"
```

Comment:
Duplicate unused variable, only actually used by reference to host_url below in playbooks.